packages/uui-button-inline-create/lib/uui-button-inline-create.element.ts

```diff
@@ -37,8 +38,26 @@ export class UUIButtonInlineCreateElement extends LitElement {
   @property({ type: Boolean, reflect: true })
   vertical = false;
 
+  /**
+   * Set an href, this will turns the button into an anchor tag.
+   * @type {string}
+   * @attr
+   * @default undefined
+   */
+  @property({ type: String })
+  public href?: string;
+
+  /**
+   * Set an anchor tag target, only used when using href.
+   * @type {string}
+   * @attr
+   * @default undefined
+   */
+  @property({ type: String })
+  public target?: '_blank' | '_parent' | '_self' | '_top';
+
   private _onMouseMove(e: MouseEvent) {
-    this._position = this.vertical ? e.offsetY : e.offsetX;
+    this._position = (this.vertical ? e.offsetY : e.offsetX) + 5;
```

Comment:
Why 5 exactly? 😎

Reply:
Yeah, this is counter offsetting what was offset via CSS.

I have removed this, and adjusted the CSS as the cursor was a bit too off. Now it looks like this, moving the button a bit off so you can see the plus, but still it feels like the mouse is on top. (Before is was just on the edge)
![IMG_0091](https://github.com/umbraco/Umbraco.UI/assets/6791648/8b93493d-4ac8-4380-9baf-931b25866ef5)